cob_hardware_config/robots/common/raw_mini_base_controller.yaml

```diff
@@ -0,0 +1,19 @@
+## joint_names
+joint_names: [fr_caster_rotation_joint, fl_caster_rotation_joint, br_caster_rotation_joint, bl_caster_rotation_joint, fr_caster_r_wheel_joint, fl_caster_r_wheel_joint, br_caster_r_wheel_joint, bl_caster_r_wheel_joint]
```

Comment:
this seems like a leftover from raw --> the whole file can be removed until we transition to ros-control